decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -600,6 +600,12 @@ def generate_localized_title
         some_extra_data: "1"
       }
     end
+
+    trait :user_group_mentioned_notification do
```

Comment:
I'm not so sure about having a trait only for this, specially if it's only needed for one spec. How about moving this to the spec itself?

Reply:
Done!